Components/Avatar/avatar.css

```diff
@@ -0,0 +1,23 @@
+.avatar {
+  width: var(--avatar-size);
+  border: var(--avatar-border);
+  border-radius: 50%;
+  /* margin: 2.4rem 0.5rem; */
+  margin-top: 1.2rem;
+}
+
+.avatar.avatar-extra-large {
+  width: 4em;
+}
+
+.avatar.avatar-large {
+  width: 3em;
+}
+
+.avatar.avatar-small {
+  width: 1.5em;
+}
+
+.avatar.avatar-extra-small {
+  width: 1em;
+}
```

Comment:
You can use avatar-xl/lg/sm/xs here for shorter class names. Rest looks good to me.